platform/throughput-report/index.md

```diff
@@ -8,15 +8,18 @@ related:
   - servicepulse/usage-config
 ---
 
-The Particular Service Platform is licensed based on the number of NServiceBus endpoints and the message throughput of those endpoints. In order to determine these values, a monitor needs to be run over the transport broker for a representative period of time (24 hours) and a report generated from this data to be sent to Particular.
+The Particular Service Platform is licensed based on the number of NServiceBus endpoints and the message throughput of those endpoints. In order to determine these values, a process needs to be run that queries the transport broker for up to 24 hours and a report generated from this data to be sent to Particular.
```

Comment:
This was edited to both:
- remove "monitor", which gets replaced with "process", to me "monitor" is strange and conflicts with the monitoring instance.
- update the 24 hours, to "up to", becasue there is no need to run it for 24 hours for brokers that contain historical data, eg Azure and AWS